.github/workflows/build-package-docs.yaml

```diff
@@ -122,6 +122,27 @@ jobs:
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
 
+  notify-build-failures:
+    if: failure()
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    env:
+      ROOM_URL: https://ansible-accounts.ems.host/_matrix/client/v3/rooms/!HJtetIFWYEIDBOXxFE:libera.chat/send/m.room.message
+      FAIL_MESSAGE: >-
+          This is a notification from the community package publishing workflow.
+          The community package docs build has failed.
+          @orandon @samccann
```

Comment:
```suggestion
          @oranod @samccann
```

Reply:
Just to make things confusing my matrix id is different to my username:

![image](https://github.com/ansible/ansible-documentation/assets/32749632/8b0dfb30-7b9e-4df7-8813-f73c220237c4)

Reply:
I realized this when I created the matrix account but never went back and changed it. It works when I get pinged by my username but I'm actually joined to the room as `orandon`.

Reply:
hah! TIL  oranod and his evil twin... orandon!